benefits/oauth/client.py

```diff
@@ -1,36 +1,52 @@
-import logging
+"""
+The oauth application: helpers for working with OAuth clients.
+"""
 
-from django.conf import settings
+import logging
 
 from authlib.integrations.django_client import OAuth
 
+from benefits.core.models import AuthProvider
+
 
 logger = logging.getLogger(__name__)
 
+oauth = OAuth()
 
-_OPENID_SCOPE = "openid"
 
+def client_kwargs(scope=None):
+    """
+    Generate the OpenID Connect client_kwargs, with optional extra scope(s).
 
-def instance(scope=None):
+    `scope` should be a space-separated list of scopes to add.
     """
-    Get an OAuth client instance using the OAUTH_CLIENT_NAME setting.
+    scopes = ["openid", scope] if scope else ["openid"]
+    return {"code_challenge_method": "S256", "scope": " ".join(scopes)}
+
 
-    Optionally configure with a (space-separated) scope.
+def server_metadata_url(authority):
```

Comment:
Same comment as for the function above this one: Would this function ever be used outside of this module? If not, I suggest prefixing with a `_`

Reply:
👍